.github/workflows/test.yml

```diff
@@ -723,7 +723,7 @@ jobs:
                   COVERAGE_THRESHOLD_LINES: 99.33
                   COVERAGE_THRESHOLD_FUNCTIONS: 99
                   COVERAGE_THRESHOLD_STATEMENTS: 99.35
-                  COVERAGE_THRESHOLD_BRANCHES: 98.72
+                  COVERAGE_THRESHOLD_BRANCHES: 98.71
```

Comment:
It is complaining because I removed a single `if` check:

Develop: https://github.com/ArdentHQ/arkvault/actions/runs/11256250466/job/31297689337

Current: https://github.com/ArdentHQ/arkvault/actions/runs/11256973861/job/31300126644